IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRail/Entities/IfcElectricFlowTreatmentDeviceType/Documentation.md

```diff
@@ -1,9 +1,9 @@
-The flow treatment device type \*\*IfcElectricFlowTreatmentDeviceType\*\* defines commonly shared information for occurrences of mobile telecommunications appliances. The set of shared information may include:
+The flow treatment device type _IfcElectricFlowTreatmentDeviceType_ defines commonly shared information for occurrences of mobile telecommunications appliances. The set of shared information may include:
 
 \* common properties with shared property sets
 \* common representations
 \* common materials
 \* common composition of elements
 \* common ports
```

Comment:
I'm unsure, but wouldn't this fail as well? Or is this handled in another PR that I'm just unaware of?